app/src/main/res/values/strings.xml

```diff
@@ -422,6 +422,7 @@
     <!-- Content Description for session item share button -->
     <string name="content_description_session_share">Share session</string>
 
+    <string name="paste">Paste</string>
```

Comment:
Take out this string and use `browser_toolbar_long_press_popup_paste` pls

Reply:
🤦‍♂